openhtf/io/proto/test_runs.proto

```diff
@@ -93,61 +210,96 @@ message TestRun {
   // the test or otherwise responsible for the data.
   optional string operator_name = 22;
 
-  // Not supported by OpenHTF
```

Comment:
this is the one delta i'd like to keep, even though i know it creates some divergence between the two copies.  I'd be okay with keeping fields not deprecated if you want to include a comment at the top of the file indicating that this is a mirror of a google internal file and not all fields are used/supported by OpenHTF.